samples/ComputerVision/ExtractText/Program.cs

```diff
@@ -42,12 +42,12 @@ public static async Task RunAsync(string endpoint, string key)
             string remoteImageUrl = "https://github.com/Azure-Samples/cognitive-services-sample-data-files/raw/master/ComputerVision/Images/printed_text.jpg";
 
             Console.WriteLine("Text being extracted ...");
-            await ExtractTextFromStreamAsync(computerVision, localImagePath, numberOfCharsInOperationId, TextRecognitionMode.Handwritten); // if localImagePath points to an image with printed text, use TextRecognitionMode.Printed 
-            await ExtractTextFromUrlAsync(computerVision, remoteImageUrl, numberOfCharsInOperationId, TextRecognitionMode.Printed);
+            await ExtractTextFromStreamAsync(computerVision, localImagePath, numberOfCharsInOperationId); 
+            await ExtractTextFromUrlAsync(computerVision, remoteImageUrl, numberOfCharsInOperationId);
         }
 
         // Read text from a remote image
-        private static async Task ExtractTextFromUrlAsync(ComputerVisionClient computerVision, string imageUrl, int numberOfCharsInOperationId, TextRecognitionMode textRecognitionMode)
+        private static async Task ExtractTextFromUrlAsync(ComputerVisionClient computerVision, string imageUrl, int numberOfCharsInOperationId)
```

Comment:
Rename to BatchReadFileFromUrlAsync. Please also rename the name of the project and the directory name to BatchReadFile.